modules/custom/az_global_footer/data/az_global_footer.json

```
@@ -247,7 +247,7 @@
       "parent_link_id": "0",
       "menu": "az-footer-topics",
       "title": "Global Engagement",
```

Update from Matt Littler in Arizona International.
```suggestion
      "title": "International Engagement",
```